home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/address/Address.java

```diff
@@ -1,10 +1,11 @@
-package com.softserveinc.ita.homeproject.homedata.entity;
```

Comment:
I think that it isn't worth to create packages containing only one entity. It seems to me better to create packages with a more general purpose, e.g.:
- package "cooperation_element" - includes Apartment, Cooperation, House, Ownership, User;
- package "authorities" - classes Permission, Role, RoleEnum, UserCooperation.
The “question” package must be fully transferred to the “poll” package.
And to leave the Address, News and BaseEntity classes in the root of the "entity" package.